modules/payment_reference/src/Plugin/Field/FieldType/PaymentReference.php

```diff
@@ -38,7 +38,6 @@ class PaymentReference extends ConfigurableEntityReferenceItem {
    */
   public static function defaultStorageSettings() {
     return [
-      'target_bundle' => 'payment_reference',
```

Comment:
We should re-add this as a field-level handler setting, can you do a new PR for that? Merging for now since this prevents a lot of tests from working.